spec/models/npq_application_spec.rb

```diff
@@ -353,10 +353,36 @@ def subject.transient_previously_funded
   end
 
   describe "validations" do
-    context "when validating funding eligibility" do
-      let(:participant_declaration) { create(:npq_participant_declaration, state: participant_declaration_state) }
-      let(:npq_application) { participant_declaration.participant_profile.npq_application }
+    let(:participant_declaration) { create(:npq_participant_declaration, state: participant_declaration_state) }
+    let(:npq_application) { participant_declaration.participant_profile.npq_application }
+
+    describe "funding eligibility with funded places" do
+      let(:valid_state) { "submitted" }
+      let(:participant_declaration_state) { valid_state }
+
+      it "is not valid if `eligible_for_funding` is false and `funded_ place` is true" do
+        npq_application.eligible_for_funding = false
+        npq_application.funded_place = true
+
+        expect(npq_application.valid?(:admin)).to eq(false)
+        expect(npq_application.valid?).to eq(true)
+      end
+
+      it "is valid if `eligible_for_funding` is false and `funded_ place` is false" do
+        npq_application.eligible_for_funding = false
+        npq_application.funded_place = false
 
+        expect(npq_application.valid?(:admin)).to eq(true)
+      end
+      it "is valid if `eligible_for_funding` is false and `funded_ place` is nil" do
```

Comment:
could we add a test for true/true, and true/nil?

Reply:
Addressed in https://github.com/DFE-Digital/early-careers-framework/pull/4863/commits/f5745404f8659707312308a12abfe78ce453e83a